src/Symfony/Component/Messenger/Bridge/Doctrine/Transport/Connection.php

```diff
@@ -399,14 +406,12 @@ private function executeQuery(string $sql, array $parameters = [], array $types
         try {
```

Comment:
I am not sure how to fix the above psalm errors. I didn't change this code so the error I think already exists. Is this because of the psalm baseline?

Reply:
:brain: wins over psalm :)